server/src/test/java/org/elasticsearch/search/aggregations/metrics/MinAggregatorTests.java

```diff
@@ -889,12 +890,17 @@ private Aggregator mockAggregator() {
         return config;
     }
 
-    private ValuesSourceConfig<ValuesSource.Numeric> mockDateValuesSourceConfig(String fieldName, boolean indexed) {
+    private ValuesSourceConfig<ValuesSource.Numeric> mockDateValuesSourceConfig(String fieldName, boolean indexed,
```

Comment:
There is a *ton* more ceremony here because `resolution` is only set when you call `build`. Bleh.